at_client/src/main/java/org/atsign/client/cli/Register.java

```diff
@@ -13,90 +14,138 @@
  */
 public class Register {
     public static void main(String[] args) throws Exception {
-        if (args.length != 1) {
+
+        String apiKey = "";
+        String email = "";
+        boolean isRegistrarV3 = false;
+
+        if (args.length == 2 && args[0].equals("-e")) {
```

Comment:
can we use ArgParser library to parse the command line args

Reply:
@VJag had previously done some research and selected a library

Reply:
I talked to @VJag  and the name of the library is "pico_cli". Working on integrating that into the register CLI